services/utils/persistence/src/test/java/org/eclipse/ditto/services/utils/persistence/mongo/config/DefaultOptionsConfigTest.java

```diff
@@ -49,7 +50,9 @@ public void initMongoDbConfig() {
     public void assertImmutability() {
         assertInstancesOf(DefaultOptionsConfig.class,
                 areImmutable(),
-                provided(ReadPreference.class, ReadConcern.class, WriteConcern.class).areAlsoImmutable());
+                provided(ReadPreference.class, ReadConcern.class, WriteConcern.class).areAlsoImmutable(),
+                assumingFields("extraUriOptions")
+                        .areSafelyCopiedUnmodifiableCollectionsWithImmutableElements());
```

Comment:
This assumption seems not to be correct. The map stored in DefaultOptionsConfig seems to be modfiable (and is provided by a getter)

Reply:
You're right 👍